rules/allied-vehicles.yaml

```diff
@@ -61,8 +61,7 @@ cmin:
 		HarvestVoice: Move
 		DeliverVoice: Move
 	ChronoResourceDelivery:
-		WarpInSequence: # unused
```

Comment:
I'm pretty sure the chrono miner had separate warp in/warp out visuals. Maybe I'm wrong, I'll check.